src/events.js

```diff
@@ -2043,18 +2037,10 @@ const btnsClicked = async (connectId, formData) => {
     if (confirmVal === "cancel") return;
 
     formData['820476880'] = collectionID;
-    formData['650516960'] = getWorflow() === 'research' ? 534621077 : 664882224;
+    formData['650516960'] = getWorkflow() === 'research' ? 534621077 : 664882224;
     formData['Connect_ID'] = parseInt(document.getElementById('specimenLinkForm').dataset.connectId);
     formData['token'] = document.getElementById('specimenLinkForm').dataset.participantToken;
     
-    let bloodAccessionId = formData?.['646899796'];
-    if (bloodAccessionId) {
-        formData['646899796'] = bloodAccessionId;
-    }
```

Comment:
Where did this logic go?

Reply:
It goes nowhere. It's like `if (a) { a=a; }`. This is why I removed it.

Reply:
Oops, I see that now... I misread what this was doing.. Thanks!